### PR TITLE
Time float, increase of amplitude precision

### PR DESCRIPTION
**Pull request by AntSimi**

### CODE REVIEW

**Review by CoriPegliasco — CHANGES_REQUESTED** [review]
English modifs : 
In CHANGELOG.rst
- Now time allows second precision (instead of daily precision) in storage on uint32 from 01/01/1950 to 01/01/2086
  New identifications are produced with this type, old files could still be loaded.
  If you use old identifications for trtacking use the `--unraw` option to unpack old times and store data with the new format.
- Now amplitude is stored with .1 mm of precision (instead of 1 mm), same advice than time.

Changes in doc/run_tracking.rst
L8 : Before tracking, ...
L10 : Before tracking, display some identification files. You will learn a lot.
L27: Number of consecutive timesteps with missing detection allowed
L29 : Minimal number of timesteps to be considered as a long trajectory
L70 : same as L27
L72 : same as L 29

In share/tracking.yaml
L2 : Files produced ...
L4 : Path to save outputs
L7 : Minimal number of timesteps to be considered as a long trajectory
L9 : Number of consecutive timesteps with missing detection allowed

Questions : 
In src/py_eddy_tracker/appli/eddies.py
L284 : why the exception is "several **days** steps"?

I think you know better than me what needed change to take into account the precision changes :)

**Review by CoriPegliasco — APPROVED** [review]

File: src/scripts/EddyTranslate [hunk 1]
```diff
@@ -16,6 +16,7 @@ def id_parser():
     )
     parser.add_argument("filename_in")
     parser.add_argument("filename_out")
+    parser.add_argument("--unraw", action="store_true", help="Load unraw data")
```
Comment (ludwigVonKoopa) [inline src/scripts/EddyTranslate:19]: can you please elaborate on what are "raw" data in documentation ?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -10,6 +10,10 @@ and this project adheres to `Semantic Versioning <https://semver.org/spec/v2.0.0
 ------------
 Changed
 ^^^^^^^
+- Now time will be allow second precision in storage on uint32 from 01/01/1950 to 01/01/2086
```
Comment (ludwigVonKoopa) [inline CHANGELOG.rst]: do you mean "time will allow second precision"  or "time precision extended from day to second" ?

**Review by ludwigVonKoopa — DISMISSED** [review]

File: src/scripts/EddyTranslate (hunk 1 above)
Comment (ludwigVonKoopa) [inline src/scripts/EddyTranslate:19]: can you please elaborate on what are "raw" data in documentation ?

**Review by ludwigVonKoopa — CHANGES_REQUESTED** [review]
some comments on the code, 

but as it is an important change, and affect a lot of module / class / functions, we should implements some tests ?

**Comment by AntSimi** [comment]
> but as it is an important change, and affect a lot of module / class / functions, we should implements some tests ?
I run tests from detection to final tracking with grid file with 3h step. For now i didn't imagine other tests.

File: CHANGELOG.rst (hunk 2 above)
Comment (AntSimi) [reply CHANGELOG.rst]: I am not sure to understand differences between this 2 sentence

Comment (CoriPegliasco) [reply CHANGELOG.rst]: It's becouse second is also 2nd I think (second decimal 0.01, even decimal is not written). "Time precision extended from day to second" remove the ambiguity

**Review by ludwigVonKoopa — APPROVED** [review]

**Pull request merged**